recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -17,3 +17,20 @@ do_install_append() {
     sed -i -e "s/\/lib\/systemd\/systemd-quotacheck/\/usr\/sbin\/quotacheck -avum/" \
           ${D}${systemd_unitdir}/system/systemd-quotacheck.service
 }
+
+quotacheck_srv = "systemd-quotacheck.service"
+quotaon_srv = "quotaon.service"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+do_install_append () {
+    install -d ${D}${sysconfdir}/systemd/system/multi-user.target.wants
+
+    sed -i -e "s/\/lib\/systemd\/systemd-quotacheck/\/usr\/sbin\/quotacheck -avum/" \
```

Comment:
this line is not needed. Also it should be in one do_install_append line

Reply:
accept